lib/metasploit/framework/login_scanner/http.rb

```diff
@@ -332,6 +334,7 @@ def create_client(opts)
             kerberos_authenticator = kerberos_authenticator_factory.call(username, password, realm)
           end
 
+          http_logger_subscriber = Rex::Proto::Http::HttpLoggerSubscriber.new(logger: framework_module)
```

Comment:
We'll either need to update the existing scanner tests to always pass in a framework_module, or make this nil safe

```suggestion
          http_subscriber = framework_module.nil? ? nil : Rex::Proto::Http::HttpLoggerSubscriber.new(logger: framework_module)
```

Reply:
I think the later is a better option, as it allows future login scanner specs to not worry about including logging as well.